lerna.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "independent",
+  "npmClient": "yarn",
+  "packages": ["packages/*"],
+  "useWorkspaces": true,
+  "command": {
+    "publish": {
+      "allowBranch": "main"
+    }
+  },
+  "local": {
+    "public": true,
+    "repo": "https://github.com/TryGhost/Ghost-CLI",
+    "scope": "@tryghost"
```

Comment:
Not sure if this will affect how the ghost-cli package gets published 🤔  hopefully it only affects new packages?

Reply:
from the looks of it this `local` config is only used by slimer-cli, which seems to only be for new packages?